### ISSUE TITLE
VoiceOver announces an extra unnamed ScrollView semantic node before the first list item

### ISSUE THREAD

**Issue by HoseinHaqiqian** [flutter/flutter#175774]
### Steps to reproduce

1. Create a new Flutter app.
2. Add a SingleChildScrollView (or CustomScrollView) with several children.
3. Run the app on iOS.
4. Turn on VoiceOver.
5. Navigate into the screen and perform the first swipe right (next item).
6. Observe the extra announcement before the first list element.

### Expected results

Focus should land directly on (and VoiceOver should announce) the first actionable child item in the scroll view  without an extra unnamed / unhelpful semantic node being announced.

### Actual results

First swipe after entering the screen: VoiceOver announces an item corresponding to the scroll container itself instead of the first list child.
The announcement text is not helpful (e.g. “Item Description Unknown”).
Only after an additional swipe does VoiceOver announce the real first list item.

### Code sample

<details open><summary>Code sample</summary>

```dart
body: Center(
        child: SingleChildScrollView(
          child: Column(
            mainAxisAlignment: MainAxisAlignment.center,
            children: List.generate(40, (index) => Text("Text at line $index")),
          ),
        ),
      ),
```

</details>

### Screenshots or Video

<details open>
<summary>Screenshots / Video demonstration</summary>

Without a ScrollView:

https://github.com/user-attachments/assets/a6647c55-13ca-4acf-94d2-350f857a7be6

With a ScrollView:

https://github.com/user-attachments/assets/a16f4c03-e34a-434b-ad3c-044263257c2a

[Upload media here]

</details>

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

### Flutter Doctor output

<details open><summary>Doctor output</summary>

```console
[✓] Flutter (Channel stable, 3.35.4, on macOS 26.0 25A354 darwin-arm64, locale en-CA)
[!] Android toolchain - develop for Android devices (Android SDK version 36.0.0)
[✓] Xcode - develop for iOS and macOS (Xcode 26.0)
[✓] Chrome - develop for the web
[✓] Android Studio (version 2024.2)
[✓] VS Code (version 1.104.0)
[✓] Connected device (4 available)
[✓] Network resources
```

</details>

**Comment by darshankawar** [comment]
@HoseinHaqiqian There's a similar [issue](https://github.com/flutter/flutter/issues/164483) open describing your case, please follow-up in it for further updates.
